app/models/service_day.rb

```diff
@@ -23,6 +23,7 @@ class ServiceDay < UuidApplicationRecord
   validates :absence_type, inclusion: { in: ABSENCE_TYPES }, allow_nil: true
   validates :date, date_time_param: true, presence: true
   validate :prevent_creation_of_absence_without_schedule
+  validates :child, uniqueness: { scope: :date }
```

Comment:
Is this pulled in from the other PR for uniqueness indexing?  If so, we should remove it from this PR